rollup.config.js

```diff
@@ -21,32 +25,29 @@ const globals = external.reduce((prev, current) => {
 const defaultConfig = {
   input: pkg.main,
   output: {
-    file: path.resolve(__dirname, pkg.lib),
-    format: 'cjs',
+    file: path.join(__dirname, pkg.lib),
+    format: 'es',
```

Comment:
why?

Reply:
当前代码添加  "type": "module"改为esm模式。
如果输出cjs模式则会出现：`ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and 'D:\html\tdesign-starter-cli\package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.`
![image](https://github.com/Tencent/tdesign-starter-cli/assets/81673017/d9903d2f-c4d2-404f-a491-9770866af434)
![image](https://github.com/Tencent/tdesign-starter-cli/assets/81673017/98e56287-ca11-4e6a-a1dd-7fca8f3961f5)

Reply:
但是这个是运行在node的

Reply:
🤔 node=>16.xx 可以使用.